notebooks/model_tutorial/1a_model_training_sensorium.ipynb

```diff
@@ -8,7 +8,7 @@
     }
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Our LN model has the same architecture as our CNN model (a convolutional core followed by a gaussian readout)
</pre><p>Is there a reason this is a comment in the cell and not a markdown cell before this cell? If there is no specific reason, could you put the text into a mardown cell before this cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
fixed